skyrl-tx/tx/tinker/engine.py

```diff
@@ -110,6 +111,7 @@ def __init__(
 
             # Split model into LoRA and non-LoRA parameters
             self.graphdef, self.lora_params, self.non_lora_params = nnx.split(self.model, self.model.is_lora_param, ...)
+            update_adapter_config(self.model, adapter_index=0, lora_rank=1, lora_alpha=1.0)
```

Comment:
This is great and makes a lot of sense. Before merging it, I would like to try a small variation of this that doesn't need to load the extra all-zeros lora adapter onto adapter_index 0 -- we can just use a non-existent adapter_index -1, and then in https://github.com/NovaSky-AI/SkyRL/blob/main/skyrl-tx/tx/layers/lora.py#L78 pass those activations through without putting them into the ragged_dot. It means we do not need to waste a lora adapter on all zeros, in OSS people typically operate close to the memory limit so it would be nice to be able to save that memory. Also that solution would be more efficient in terms of computation. It is possible that the code for this will be very simple, but if it is complex, let's keep the additional zeros adapter :)